packages/sit-onyx/src/i18n/index.ts

```diff
@@ -87,11 +79,13 @@ const createI18n = (options: ProvideI18nOptions = {}) => {
    * Current locale.
    * @default "en-US"
    */
-  const locale = computed(() => unref(options?.locale) ?? "en-US");
+  const locale = toRef(options?.locale ?? "en-US");
```

Comment:
```suggestion
  const locale = toRef(options?.locale ?? "en-US");
```

Shouldn't we make the locale readonly? So we don't unintentionally change it internally